File: 3_upside_down_triangle/upside_down_triangle.jl
function upside_down_triangle(n::Int)::Nothing
    n >= 1 || error("Value must be >= 1")

    for i=0:n
        println("*"^(n-i))
    end
end

print("Enter a number for the base: ")
n = readline()
n = parse(Int, n)
upside_down_triangle(n)

File: 3_upside_down_triangle/upside_down_triangle.cpp
#include <iostream>
#include <stdexcept>

using namespace std;

void upside_down_triangle(int n){
    if(n < 1){
        throw invalid_argument("Value must be >= 1");
    };

    for(int i=0; i < n; i++){
        for(int j=1; j <= n-i; j++){
            cout << "*";
        };
        cout << endl;
    };
};

int main(){

    cout << "Enter a number for the base: ";
    int n;
    cin >> n;
    upside_down_triangle(n);
}